src/pages/events/summit-event-list-page.js

```diff
@@ -125,7 +126,8 @@ class SummitEventListPage extends React.Component {
                 all_companies: [],
                 submission_status_filter: []
             },
-            selectedColumns: []
+            selectedColumns: [],
+            searchTerm: props.term
```

Comment:
does not use the state, bc its clear the search criteria once u enter to an activity to edit it